docs/OV_Converter_UG/prepare_model/convert_model/MO_OVC_transition.md

```diff
@@ -624,6 +624,175 @@ Here is the guide to transition from legacy model preprocessing to new API prepr
 
             - Not available in OVC tool. Please check Python API.
 
+Input and output cut
+##########################
+
+In new conversion API input and output model cut are no longer available. It is recommended to perform cut in original framework instead.
+Below are examples of input and output cut for TF1, TF saved model and ONNX formats in legacy conversion API with example of how to perform identical cut in original framework.
+For PyTorch, TF Keras and PaddlePaddle it is recommended to change original model code to perform model cut. 
+
+``TF frozen .pb format / frozen ft.Graph / tf.GraphDef``
+########################################################
+
+.. tab-set::
+    .. list-table::
+       :header-rows: 1
+
+       * - Legacy API
+         - New API
+       * - .. code-block:: py
+              :force:
+
+              import openvino as ov
+              import openvino.tools.mo as mo
+               
+              import tensorflow as tf
+              from tensorflow.python.tools.strip_unused_lib import strip_unused
+               
+               
+              def load_graph(model_path):
+                  graph_def = tf.compat.v1.GraphDef()
+                  with open(model_path, "rb") as f:
+                      graph_def.ParseFromString(f.read())
+                  with tf.compat.v1.Graph().as_default() as graph:
+                      tf.graph_util.import_graph_def(graph_def, name="")
+                      return graph
+               
+               
+              # Load TF model
+              graph = load_graph("/path_to_model/HugeCTR.pb")
+               
+              # Convert the model with input and output cut
+              input = "concat"
+              output = "MatVec_3/Squeeze"
+              ov_model = mo.convert_model(graph, input=(input, [-1, -1]), output=output)
+               
+              # Compile the model
+              compiled_model = ov.compile_model(ov_model)
+
+
+         - .. code-block:: py
+              :force:
+
+              import openvino as ov
+              import tensorflow as tf
+               
+              from tensorflow.python.tools.strip_unused_lib import strip_unused
+               
+              def load_graph(model_path):
+                  graph_def = tf.compat.v1.GraphDef()
+                  with open(model_path, "rb") as f:
+                      graph_def.ParseFromString(f.read())
+                  with tf.compat.v1.Graph().as_default() as graph:
+                      tf.graph_util.import_graph_def(graph_def, name="")
+                      return graph
+               
+              # Load TF model
+              graph = load_graph("/home/panas/HugeCTR.pb")
+              
+              # Cut the model
+              input = "concat"
+              output = "MatVec_3/Squeeze"
```

Comment:
I think that cutting should be by tensors so it should contain `:0`

Reply:
`strip_unused()` works only with node names. Adding `:0` leads to error:
ValueError: Name 'concat:0' appears to refer to a Tensor, not an Operation.